src/components/VHeader/VFilterButton.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <VButton
+    :variant="buttonVariant"
+    class="self-center"
+    :pressed="pressed"
+    aria-controls="filter-sidebar"
+    @click="toggleFilters"
+  >
+    <VIcon
+      v-show="showIcon"
+      :icon-path="filterIcon"
+      :class="{ 'me-2': iconNeedsPadding }"
```

Comment:
This padding is displaying on middle breakpoints, where the text label is no longer visible:

<img width="316" alt="CleanShot 2021-12-09 at 13 07 22@2x" src="https://user-images.githubusercontent.com/6351754/145451642-c9357545-4502-4b05-87b9-d941f92a565c.png">